README.md

```diff
@@ -4,20 +4,35 @@ Cryptic World is a Project that uses a Crypto API, to fetch data about Crypto Cu
 
 # Live demo
 
-not yet...
+here is a live demo of the site
 
-# Install
+https://www.loom.com/share/0a83e451b9db469e8c2fd6ac5f220fbf
+
+# Prerequisites
+
+In order to run this project you need:
 
-type npm install
+Node.js
+NPM Package Manager
 
-# start
+# Setup
 
-to start the project type npm start
+Clone this repository to your desired folder:
+
+git clone https://github.com/david63011/React-Capstone-Crypto-Market.git
+
+# Install
+
+to install this program run npm install
 
 # test
 
 to write tests type npm run test
 
+# Deployment
+
+you can deploy this project by using npm run deply
```

Comment:
- [ ] kindly use the microverse [readme](https://github.com/microverseinc/readme-template) template and also add the link to your project

Reply:
ok fixed